Yuval/patchset15

yuval-lb opened the pull request:
gorka comments on patchset 15-16

yuval-lb committed:
additionl driver opts

yuval-lb committed:
remove dsc check on setup

yuval-lb committed:
project name already assigned

yuval-lb committed:
else is redundant since we return in the "if" block

muliby-lb commented on cinder/volume/drivers/lightos.py:
```diff
@@ -1435,10 +1435,9 @@ def terminate_connection(self, volume, connector, **kwargs):
-                msg = 'Connector (%s) did not return a hostnqn, aborting' % (
-                    connector,)
-                raise exception.VolumeBackendAPIException(message=msg)
+            msg = 'Connector (%s) did not return a hostnqn, aborting' % (
```
one empty line above this please

yuval-lb committed:
update doc typo

yuval-lb committed:
get_driver_options - remove fields

volume_name_template and snapshot_name_template cant be added to _get_oslo_driver_opts
they are not in the driver.VolumeDriver cfg

ronen-lb commented on doc/source/configuration/block-storage/drivers/lightbits-lightos-driver.rst:
```diff
@@ -138,4 +138,4 @@ Lightbits LightOS Cinder driver.
    :config-target: Lightbits LightOS
 
-   cinder.volume.drivers.kioxia.kumoscale
+   cinder.volume.drivers.lightos
```
+1

tomerg-lb commented:
@yuval-lb can this be closed?
